packages/loot-core/src/server/accounts/sync.ts

```diff
@@ -737,8 +737,12 @@ export async function addTransactions(
   await createNewPayees(payeesToCreate, added);
 
   let newTransactions;
-  if (runTransfers) {
-    let res = await batchUpdateTransactions({ added });
+  if (runTransfers || learnCategories) {
+    let res = await batchUpdateTransactions({
+      added,
+      learnCategories: learnCategories,
+      runTransfers: runTransfers,
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
      learnCategories,
      runTransfers,
```